doc/userguide/src/ExecutingTestCases/BasicUsage.rst

```diff
@@ -23,21 +23,19 @@ Synopsis
 ::
 
     robot [options] data_sources
-    python|jython|ipy|pypy -m robot [options] data_sources
-    python|jython|ipy|pypy path/to/robot/ [options] data_sources
-    java -jar robotframework.jar [options] data_sources
+    python|pypy -m robot [options] data_sources
+    python|pypy path/to/robot/ [options] data_sources
```

Comment:
I guess we could use just `python`. Not everyone knows about PyPy and including `pypy` can be confusing. People who know and use it also know they can use `pypy` instead of `python` anyway.